pkg/modprovider/server.go

```diff
@@ -129,61 +130,99 @@ func dirExists(path string) bool {
 	return false
 }
 
+func extractConfigParamFromArgs(args []string) ([]string, string, bool) {
+	for i, arg := range args {
+		if (arg == "--config" || arg == "-c") && i+1 < len(args) {
+			return args[:i], args[i+1], true
```

Comment:
The positional arg has to be the last arg right? E..g. not `--config c.cfg vpc`

Reply:
Yes, that is the implicit contract here